src/timeMachine/reducers/index.ts

```diff
@@ -694,7 +694,10 @@ export const timeMachineReducer = (
     }
 
     case 'SET_BACKGROUND_THRESHOLD_COLORING': {
-      const viewColors = state.view.properties.colors as Color[]
+      let viewColors = []
+      if ('colors' in state.view.properties) {
+        viewColors = state.view.properties.colors as Color[]
+      }
```

Comment:
grabbed this from here: https://github.com/influxdata/ui/pull/2415/files#r696188499